common/src/main/java/com/sequenceiq/cloudbreak/telemetry/TelemetryUpgradeConfiguration.java

```diff
@@ -15,8 +15,6 @@ public class TelemetryUpgradeConfiguration {
 
     private TelemetryComponentUpgradeConfiguration meteringAgent;
 
-    private TelemetryComponentUpgradeConfiguration cdpVmAgent;
```

Comment:
sorry one thing that i have missed, this config is referenced here as well: https://github.com/hortonworks/cloudbreak/blob/master/common/src/main/resources/common-config.yml#L88